en/docs/install-and-setup/setup/security/configuring-transport-level-security.md

```diff
@@ -82,7 +83,42 @@ By default, all WSO2 products pass "WSO2 Carbon Server" as the server value in H
     server="WSO2 Carbon Server"
     ```
 
-!!! info
-    See the [Security Guidelines for Production Deployment](https://docs.wso2.com/display/ADMIN44x/Security+Guidelines+for+Production+Deployment) for the full list of security-related recommendations for WSO2 products.
+## Disabling HTTP Transports
+
+!!! Note
+    It is recommended to disable the HTTP transport in an API Manager production setup. Using the `Bearer` token over HTTP is a violation of the OAuth specification and can lead to security vulnerabilities.
+
+API Manager has two HTTP transports.
+
+-   Passthru (API Traffic) Transport
+-   Servlet (UI Traffic and Admin service access) Transport
+
+See the instructions given below to disable these transports.
+
+### Disabling Passthrough Transport
```

Comment:
@tharindu1st 
Could you confirm if this is accurate? I can see the passthrough config section in the Config Catalog[1], however, it conflicts with the toml headers we have here in this doc.

[1] https://apim.docs.wso2.com/en/latest/reference/config-catalog/#https-pass-through-transport-non-blocking-mode

Reply:
default value is true

Reply:
That means, to disable we have to set this value to 'false' as given in the doc.
I take it that the doc is correct in that case.
Ack.